app/views/categories/index.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="categories_sect">
```

Comment:
- [ ] [OPTIONAL] Kindly provide a way for deleting `Categories` and `Transactions`.